Orchestrator/README.md

```diff
@@ -1,20 +1,177 @@
 
+- Improved classification accuracy
+- Higher resilience to data imbalance across your LUIS and QnA Maker authoring data.
+- Ability to correctly dispatch from relatively little authoring data.
 
-## References
+**Intent Recognizer**: You can use Orchestrator as an intent recognizer with [Adaptive dialogs][6]. Using the same approach as in the dispatch scenario above in order to route to responses within your bot logic.
 
+**Entity Extraction** is not supported yet.  It is on the planning roadmap to add entity extraction in the future.
 
+## Authoring Experience
 
-- [Tech overview](https://github.com/microsoft/BotBuilder-Samples/blob/main/experimental/orchestrator/docs/Overview.md)
-- [API reference](https://github.com/microsoft/BotBuilder-Samples/blob/main/experimental/orchestrator/docs/API_reference.md)
-- [Roadmap](https://github.com/microsoft/BotBuilder-Samples/blob/main/experimental/orchestrator/docs/API_reference.md#Roadmap)
-- [BF CLI Orchestrator plugin](https://github.com/microsoft/botframework-cli/tree/beta/packages/orchestrator )
-- [Natural Language Representation](https://aka.ms/NLRModels) base models
+Orchestrator can be used in different development environments:
 
+* **Code First**: Orchestrator can be integrated into your code project by replacing LUIS for intent recognition such as for skill delegation or dispatching to subsequent language understanding services.  See [Runtime Integration](#runtime-integration) section for more.
+* [Bot Framework Composer][19]: Orchestrator can be selected as a recognizer within Bot Framework Composer. At this point there are limitations to using Orchestrator in Composer primarily around importing of existing models and tuning up recognition performance (* currently available only if the feature flag is enabled with Composer).
 
+Thus, use of [BF command line tool][7] to prepare and optimize the model for your domain is required in most, if not all, use cases.   To illustrate the workflow, here is a sample the end to end authoring experience:
 
+<p align="center">
+  <img width="350" src="./docs/media/authoring.png" />
+</p>
+
+### Prepare
+
+* Pre-requisite: Install [BF CLI Orchestrator plugin][11] first.
+
+1. Author Intent-utterances example based .lu definition referred to as a *label file* using the Language Understanding practices as described in [Language Understanding][2] for dispatch (e.g. author .lu file or within the [Composer][3] GUI experience). 
+   * Alternatively, [export][8] your LUIS application and [convert][9] to .lu format or [export][10] your QnA Maker KB to .qna format.
+   * See also the [.lu file format][21] to author a .lu file from scratch. 
+2. Download Natural Language Representation ([NLR][20]) base Model (will be referred to as the *basemodel*) using the `bf orchestrator:basemodel:get` command. 
```

Comment:
Download Orchestrator base model using 'bf orchestrator:basemodel:get' ...  

We no longer use NLR term..